components/email.php

```diff
@@ -0,0 +1,213 @@
+<?php
+/**
+ * Manage BuddyPress Email Post Types.
+ *
+ * @since 1.6.0
+ */
+class BPCLI_Email extends BPCLI_Component {
+	/**
+	 * Create a new email post connected to an email type.
+	 *
+	 * ## OPTIONS
+	 *
+	 * [--type]
```

Comment:
This could use some improvements. `--type=<type>` 
Using [] will make it optional. So if it is required, remove it
Also, it is missing the last part, the `<content>` part.

Reply:
Thanks!  Will change this.